packages/typescript-generator/templates/typescript-package/package.json.hbs

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@pagopa/{{packageName}}",
+  "version": "0.1.0",
+  "type": "module",
+  "description": "{{description}}",
+  "repository": {
+    "type": "git",
+
+    "url": "git+https://github.com/pagopa/dx.git",
+    "directory": "packages/{{packageName}}"
+  },
+  "keywords": [],
+  "files": [ "dist" ],
+  "main": "./dist/index.js",
+  "exports": {
+    ".": { "import": "./dist/index.js" }
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@pagopa/eslint-config": "workspace:^",
+    "@tsconfig/node20": "^20.1.5",
+    "@types/node": "^20.12.2",
+    "eslint": "8.57.0",
+    "tsup": "^8.5.0",
+    "typescript": "~5.8.3",
+    "vitest": "^1.6.0"
```

Comment:
hardcoding these values is the best way to spread legacy dependencies and technical debt! (example: the latest version of `vitest` is `3.x`, I don't think we should suggest to use the `1.x`)

this section should be valued dinamically by the package manager, in a step that should happen before the code generation

example of the dep setup script:

```bash
yarn add -D @pagopa/eslint-config @tsconfig/node20 @types/node eslint@^8.0.0 typescript vitest
yarn add -DE prettier
```